backends/qualcomm/qnn_quantizer.py

```diff
@@ -116,23 +69,21 @@ def _derive_bias_qparams_fn(
     )
 
 
-def get_default_qnn_ptq_config(
-    enable_per_channel_conv_quant=False,
-) -> Tuple[QuantizationConfig, QnnQuantizerConfig]:
+def get_default_8bit_qnn_ptq_config() -> QuantizationConfig:
     extra_args: Dict[str, Any] = {"eps": 2**-12}
 
     act_quantization_spec = QuantizationSpec(
         dtype=torch.uint8,
         quant_min=0,
-        quant_max=255,
+        quant_max=torch.iinfo(torch.uint8).max,
         qscheme=torch.per_tensor_affine,
-        observer_or_fake_quant_ctr=HistogramObserver.with_args(**extra_args),
+        observer_or_fake_quant_ctr=MovingAverageMinMaxObserver.with_args(**extra_args),
```

Comment:
Thank you for pointing out. We found that the accuracy is wrong when we use Histogram observer for mobilebert. Yet MovingAverageMean observer can obtain correct result, and it works on all the other test cases. Therefore we assume it might be fine to use MovingAverageMean as our default observer.
Would you recommend  to define a specific annotation configuration for mobilebert. If so we will make the config in  examples/qualcomm/scripts/ptq_mobilebert_fine_tune.py specifically

Reply:
If it is observed to work well for other models, then I dont have much concern. But if not i would modify specificaly for mobilebert

Also, would love to understand why histogram doesnt work for mobilbert but moving average works. This is interesting observation.